packages/syft/src/syft/types/newdatetime.py

```diff
@@ -0,0 +1,45 @@
+# stdlib
+from datetime import datetime, timezone
+from functools import total_ordering
+from typing import Any
+
+# third party
+from typing_extensions import Self
+import pydantic
+
+# relative
+from ..serde.serializable import serializable
+from datetime import datetime
+
+@serializable()
+@total_ordering
+class NewDateTime(pydantic.BaseModel):
+    # id: UID | None = None  # type: ignore
+    utc_timestamp: float
+
+    # @classmethod
+    # def now(cls) -> Self:
+    #     return NewDateTime.utcnow()
+
+    @classmethod
+    def now(cls) -> Self:
+        return cls(utc_timestamp=datetime.utcnow().timestamp())
+
+    def __str__(self) -> str:
+        utc_datetime = datetime.utcfromtimestamp(self.utc_timestamp)
+        return utc_datetime.strftime("%Y-%m-%d %H:%M:%S")
+
+    def __hash__(self) -> int:
+        return hash(self.utc_timestamp)
+
+    def __sub__(self, other: "NewDateTime") -> "NewDateTime":
```

Comment:
use from typing_extensions import Self?

Reply:
Also this type departing from every other type means it can be serialized but it can never be stored, like you could never have a `NewDateTime` object as the result or input to anything else...

I do think we need the bare minimum of:
```
class SyftType:
    uid: UID
    value: Thing
    __canonicalname__: str
    __version__: int
```
Which means we can reference it, version it and then get the actual value we care about since this is essentially a versioned networked python object.

Reply:
Except that we could just wrap it in an ActionObject which... is also probably fine, but does beg the question how and why we differentiate between document objects and action objects other than applying well known math operations `__method__` for which most document objects have no well defined implementation.